Library/Formula/shml.rb

```diff
@@ -0,0 +1,16 @@
+class Shml < Formula
+  desc "Style Framework for The Terminal"
+  homepage "http://shml.xyz"
+  url "https://github.com/MaxCDN/shml/archive/1.0.4.tar.gz"
+  sha256 "583fefbbf2adf45d989b052696be0f20f695903878558a4d2e9f74255b91ed51"
+
+  def install
+    install "shml.sh"
```

Comment:
If you want to leave it in the prefix you're looking for `prefix.install` here. `install` by itself won't do anything, I believe.